src/ApiServiceProvider.php

```diff
@@ -123,15 +123,13 @@ public function add_views()
     }
 
     /**
-     * Set the paths for migrations and assets that
-     * should be published to the main application.
+     * Set the path for migrations which should
+     * be migrated by laravel. More informations:
+     * https://laravel.com/docs/5.5/packages#migrations.
      */
-    public function add_publications()
+    public function add_migrations()
```

Comment:
Should be private instead public as it will (and probably should) never be call outside of the provider.